src/components/Header/Header.js

```diff
@@ -207,64 +69,20 @@ function Header() {
     }
```

Comment:
let's put this in async method within the useEffect

Reply:
resolved in 174adae